Entity/Feature.php

```diff
@@ -57,6 +57,13 @@ class Feature
      */
     private $parent;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=512)
```

Comment:
why 512? shouldn't be nullable?

Reply:
You are right, I have to define the nullable (I did it on the AdEspresso migration and not here)... 
What's your doubt about the length? I think that maybe is too long, what do you think about it?